addons/web/static/tests/views/pivot_view_tests.js

```diff
@@ -125,6 +125,27 @@ QUnit.module("Views", (hooks) => {
                             ],
                             group_operator: "count_distinct",
                         },
+                        properties: {
+                            string: "Properties",
+                            type: "properties",
+                            definition_record: "parent_id",
+                            definition_record_field: "properties_definition",
+                            name: "properties",
+                        },
+                        parent_id: {
+                            string: "Parent",
+                            type: "many2one",
+                            relation: "partner",
+                            name: "parent_id",
+                        },
+                        properties_definition: {
+                            string: "Properties",
+                            type: "properties_definition",
+                        },
+                        "properties.my_char": {
+                            // needed for MockServer to work, see @mockReadGroup
```

Comment:
I think it would be better to define yourself the return of webReadGroup, webSearchRead in your tests instead of doing this kind of things. You have mockRPC for that. The code in mock_server is very difficult to follow and make it rely on this kind of tricks is really weird